### PR TITLE
More autoloader + jimport() stuff

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

**Comment by realityking** [comment]
Urgh, don't hit enter when the cursor is still on the title.

The two jimports() are not required anymore thanks to #604.
Moving JStringNormalize should be done now since it's a pretty new class and not many people will have it used yet.

File: tests/suite/joomla/string/JStringNormalizeTest.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * @license     GNU General Public License version 2 or later; see LICENSE
  */
 
-require_once JPATH_PLATFORM . '/joomla/string/stringnormalize.php';
```
Comment (chdemko) [inline tests/suite/joomla/string/JStringNormalizeTest.php]: See #589

Comment (realityking) [inline tests/suite/joomla/string/JStringNormalizeTest.php]: Duh. I'll take it off again ;)

File: libraries/joomla/application/application.php [hunk 2]
```diff
@@ -99,7 +99,6 @@ class JApplication extends JObject
 	 */
 	public function __construct($config = array())
 	{
-		jimport('joomla.utilities.utility');
 		jimport('joomla.error.profiler');
```
Comment (chdemko) [inline libraries/joomla/application/application.php:102]: JUitility is used in this file. Why do you want to remove the jimport ?

Comment (realityking) [inline libraries/joomla/application/application.php:102]: MAybe I'm just dense - but where? I couldn't find it. (Note that getHash was moved from JUtility to JApplication)

**Pull request merged**